homeassistant/components/imeon_inverter/strings.json

```diff
@@ -0,0 +1,186 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add Imeon inverter",
+        "data": {
+          "address": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "address": "The address of the inverter",
+          "username": "Your inverter login",
+          "password": "Your inverter password"
```

Comment:
Can we make this more clearer? because we now ask for a host and we ask for an address in the description

I would also propose to make the username password parts a bit longer `The username of your ......` thing